packages/ui/src/lib/table/Table.svelte

```
@@ -249,73 +246,48 @@ function sort(column: Column<T>): void {
-let gridTemplateColumns: string;
-$: {
-  // section and checkbox columns
+const gridTemplateColumns: string = $derived.by(() => {
   let columnWidths: string[] = ['20px'];
 
   if (row.info.selectable) {
     columnWidths.push('32px');
   }
 
-  // custom columns
   visibleColumns.map(c => c.info.width ?? '1fr').forEach(w => columnWidths.push(w));
```

Biome (2.4.16)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)